.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,24 @@
-### What change does this PR introduce?
+### Summary
 
-<!-- Explain here the changes your PR introduces and text to help us understand the context of this change. -->
+<!-- Provide a concise summary "Why are the changes needed"? 
+Include any relevant links, such as Linear tickets, Slack discussions, 
+or design documents. -->
 
-### Why was this change needed?
```

Comment:
The purpose of this section should be on the "Why". The changes made can be seen in the PR but why did we make them is very important context for the reviewer and for posterity.

Reply:
Most of our PRs don't have anything in this section, usually because the "why" is in the Linear ticket or GH issue.

Reply:
Also, I felt people would write the why in the Summary part. Right now, the questions we have are usually ignored, the what vs why is not so clear and are usually combined into a few sentences into the "What change does this PR introduce?". 
So, my thought was that it would feel more natural to write why in the Summary (its also in the description of the question) because they see they have the next question that has the space to talk about what they actually did

Reply:
In this case, we can combine them: `What changed? Why did it change?`

Reply:
OK. But I removed the Changes Made part. I don't want it to go back to being confusing with 2 too similar questions. What do you think?

Reply:
Looks good.

Reply:
Can we also use the same in the ee repo?

Reply:
of course. There is already a PR for the ee, just waited until we agreed here to change it there. Thanks. Can you please approve so I can merge this one? :)